metadata/planning/wilderness_proposal_hr1500.md

```diff
@@ -0,0 +1,69 @@
+# Title
+
+Utah Wilderness Proposal HR1500
+
+## ID
+
+6767fe71-0525-4fad-a43f-eede1f5ae5a4
+
+## Brief Summary
+
+Polygon dataset of wilderness areas proposed by Rep. Wayne Owens in 1991.
+
+## Summary
+
+This dataset contains polygons for areas proposed for wilderness designation in 1991. For datasets of current areas designated as wilderness or under consideration, please see the [Utah Wilderness Areas](https://gis.utah.gov/products/sgid/boundaries/wilderness/) and [BLM Wilderness Study Areas for Utah](https://gis.utah.gov/products/sgid/boundaries/blm-wilderness-study-areas/) layers.
+
+## Description
+
+### What is the dataset?
+
+Established in 1964, the [National Wilderness Preservation System](https://www.wilderness.org/articles/article/national-wilderness-preservation-system) allows for the protection and preservation of designated natural areas throughout the United States. This dataset contains areas proposed for wilderness designation by former Utah representative Wayne Owens in U.S. House of Representatives Bill HR1500 (1991). These areas included the Great Basin, Zion and Mojave Desert, Grand Staircase, Kaiparowits Plateau, Escalante Canyons, Henry Mountains, Dirty Devil River, Cedar Mesa Wilderness, Canyonlands, San Rafael Swell, Book Cliffs, and the Uinta Basin Wilderness regions.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available for general historical reference and cartographic purposes.
+
+### What does the dataset represent?
+
+Each polygon in this dataset indicates the geographic extent of a proposed wilderness area in Utah. Features in this dataset include the area in acres.
+
+### How was the dataset created?
+
+This dataset was created by Rep. Owen's staff in collaboration with the University of Utah Geography Department. They modified boundaries from the Utah Wilderness Coalition (UWC). The original boundaries from UWC were published in the 1990 book Wilderness at the Edge: a Citizen Proposal to Protect Utah's Canyons and Deserts.
+
+### How reliable and accurate is the dataset?
+
+This layer is intended for historic reference and does not reflect current wilderness or wilderness study area boundaries. Please reach out to [our team](https://gis.utah.gov/contact/) with questions or concerns about this dataset.
```

Comment:
Maybe something like "This layer is intended to describe lands in HR 1500 and does not reflect..."